app/kuma-dp/pkg/dataplane/metrics/merge_test.go

```diff
@@ -1,4 +1,4 @@
-package metrics_test
+package metrics
```

Comment:
Our convention is that we are trying to do the testing outside of the original package. We try no to test internal private functions but test it like the client of this package would.

However, I understand that here you had to change this for `rewriteMetricsURL`. Let's just try no to make it a common practice if possible 🙏

Reply:
I can just export the API?

Reply:
you can, but that's fine. Let's merge it